src/backend/InvenTree/users/models.py

```diff
@@ -924,3 +925,86 @@ def create_missing_rule_sets(sender, instance, **kwargs):
     As the linked RuleSet instances are saved *before* the Group, then we can now use these RuleSet values to update the group permissions.
     """
     update_group_roles(instance)
+
+
+class UserProfile(InvenTree.models.MetadataMixin):
+    """Model to store additional user profile information."""
+
+    class UserType(models.TextChoices):
+        """Enumeration for user types."""
+
+        BOT = 'bot', _('Bot')
+        INTERNAL = 'internal', _('Internal')
+        EXTERNAL = 'external', _('External')
+        GUEST = 'guest', _('Guest')
+
+    user = models.OneToOneField(User, on_delete=models.CASCADE, related_name='profile')
```

Comment:
All of these fields need translated `verbose_name` and `help_text` attributes - so that they will be auto-translated for the forms UI.

Reply:
done